README.md

```diff
@@ -1,38 +1,43 @@
 # Little Esty Shop
 
-## Background and Description
+"Little Esty Shop" is a fictitious e-commerce management platform for admins and merchants to oversee inventory, and track and fulfill customer invoices. "Little Esty Shop" is deployed via Heroku at [Little Esty Shop](https://rocky-tundra-78096.herokuapp.com/).
 
-"Little Esty Shop" is a group project that requires students to build a fictitious e-commerce platform where merchants and admins can manage inventory and fulfill customer invoices.
+### Features: Merchants
+#### Tools created to make it easy for your merchants to manage inventory and customers
+**_Merchants can track items and invoices, and easily see favorite customers._**\
+![Merchant Show Page](/app/assets/images/merchant_show.png)
 
-## Learning Goals
-- Practice designing a normalized database schema and defining model relationships
-- Utilize advanced routing techniques including namespacing to organize and group like functionality together.
-- Utilize advanced active record techniques to perform complex database queries
-- Practice consuming a public API while utilizing POROs as a way to apply OOP principles to organize code
+**_Merchants can track and change the status of their items with the click of a button._**\
+![Merchant Items Page](/app/assets/images/merchant_items_show.png)
 
-## Requirements
-- must use Rails 5.2.x
-- must use PostgreSQL
-- all code must be tested via feature tests and model tests, respectively
-- must use GitHub branching, team code reviews via GitHub comments, and github projects to track progress on user stories
-- must include a thorough README to describe the project
-- must deploy completed code to Heroku
+**_Merchants can see individual items and update their attributes._**\
+![Item Show Page](/app/assets/images/item_show.png)
 
-## Setup
+### Features: Admins
+#### Tools created to support site administrators simplify the tasks associated with managing complex commerce.
+**_Admins can track merchants, invoices, and customers from a single central dashboard._**\
+![Admin Dashboard](/app/assets/images/admin_dashboard.png)
+
+**_With a simple click of a button, admins can enable or disable their merchants._**\
+![Admin Merchants Show](/app/assets/images/admin_merchants_show.png)
+
+**_Admins can track individual invoices and statistics with ease._**\
+![Admin Invoices Show](/app/assets/images/admin_invoice_show.png)
 
-This project requires Ruby 2.7.2.
+## Shipping Changes
+If you would like to make changes to this project, please contact the owner of this repository via GitHub.
+
+## Setup
 
+This project was developed via Ruby 2.7.2, on Rails 5.2.5. To access the dev environment for this app please follow these steps:
 * Fork this repository
 * Clone your fork
 * From the command line, install gems and set up your DB:
     * `bundle`
     * `rails db:create`
-* Run the test suite with `bundle exec rspec`.
-* Run your development server with `rails s` to see the app in action.
 
```

Comment:
Maybe include the `rake csv_load:all` command as well so they'll have data to use.